src/desktop_develop.ts

```diff
@@ -227,21 +262,21 @@ class DesktopDevelopBuilder {
         // is actually the key container in the format [{{passphrase}}]=container
         this.riotSigningKeyContainer = await getSecret('riot_key_container');
 
-        this.lastBuildTimes = {};
-        this.lastFailTimes = {};
+        this.lastBuildTimes = {} as Record<Type, number>;
+        this.lastFailTimes = {} as Record<Type, number>;
```

Comment:
these casts shouldn't be needed with types declared up above

Reply:
Fixed.